cluster-autoscaler/README.md

```diff
@@ -122,4 +122,4 @@ Right now it is possible to run Cluster Autoscaler on:
 * GCE https://kubernetes.io/docs/concepts/cluster-administration/cluster-management/
 * GKE https://cloud.google.com/container-engine/docs/cluster-autoscaler
 * AWS https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/cloudprovider/aws/README.md
-* Azure https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/cloudprovider/azure/README.md
+* Azure https://docs.microsoft.com/en-us/azure/aks/autoscaler
```

Comment:
@sakthivetrivel Thanks. The new link is AKS only, while acs-engine also supports VMAS VMSS and ACS.

Could you update AKS part in file https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/cloudprovider/azure/README.md instead?